### DISCUSSION TITLE
Trying to get Global or Collection variables in flow

### DISCUSSION THREAD

**Discussion by luisfuenz** [postmanlabs/postman-flows#79] [Defect]
I'm trying to get my Global variables or my Collection variables from my Flow, but the field to select a variable is always empty and when i type a new variable in my flow, i can't keep alive to another request. 

I've got screenshots, I'm not sure why the input data is not showing the variables to select.
![image](https://user-images.githubusercontent.com/68069448/139063490-3a0dec61-f16a-4b0f-816c-619690192ed8.png)

![image](https://user-images.githubusercontent.com/68069448/139063669-b95c78de-4746-4c21-bc46-3ed3e49aa860.png)

**Comment by saswatds** [comment]
Hey, @luisfuenz the `assign variable` block can be used to define variables in the [data variable scope](https://learning.postman.com/docs/sending-requests/variables/#variable-scopes)  
If you go ahead and connect the next request in the flow, the` assign variable` will read your request and try to suggest possible options for your block.
